examples/simple-a2a/__main__.py

```
@@ -0,0 +1,52 @@
+from common.types import AgentSkill
+import click
+from common.types import AgentSkill, AgentCapabilities, AgentCard
```

Ruff (0.8.2)

Redefinition of unused `AgentSkill` from line 1

Remove definition: `AgentSkill`

(F811)